code/main/edge/stateStoreManager.brs

```diff
@@ -14,21 +14,28 @@
 function _adb_StateStoreManager() as object
 
     return {
-        _state: invalid,
+        _states: {},
 
         getStateStore: function() as object
-            _adb_logDebug("_adb_StateStoreManager::getStateStore() - returning stateStore: (" + FormatJson(m._state) + ").")
-            return m._state
-        end function,
+            payload = []
```

Comment:
The outer function now returns an array of the payload. Can we rename it to something like `getPayloadArray`?

Reply:
It always used to return array of payloads. We have just updated the way we form that.

Reply:
We use another function to create `_adb_StateStore` objects. I was initially confused, thinking that the `getStateStore` function was supposed to return an `_adb_StateStore` object. So, I would suggest renaming the function for better clarity. What do you think?

Reply:
Yes, I am also kind of lost about the naming here. let me rethink some names. Let me know if you have any suggestions.

Reply:
Updated

Reply:
Yeah, it looks better.